### ISSUE TITLE
`vncsession` should emit an error when `vncpasswd` is missing and also required

### ISSUE THREAD

**Issue by jcpunk** [TigerVNC/tigervnc#1603]
**Describe the bug**
When starting a `vncsession` if a vnc password is required by the configuration but no `vncpasswd` file exists, `vncsession` exits.

However, it does not produce an error message.

**To Reproduce**
1. setup ~/.vnc/config
2. remove ~/.vnc/passwd
3. start the vncsession service
4. See no error, but session doesn't start

**Expected behavior**
prints out something to stderr like "missing ~/.vnc/passwd or can't read ~/.vnc/passwd"

**Client (please complete the following information):**
 - OS: RHEL9
 - VNC client: TigerVNC
 - VNC client version: 1.12.0-5.el9_1.1
 - Client downloaded from: my Linux distribution

**Server (please complete the following information):**
 - OS: RHEL9
 - VNC client: TigerVNC
 - VNC client version: 1.12.0-5.el9_1.1
 - Client downloaded from: my Linux distribution
 - Server was started using: `systemctl start vncserver@:1.service`

**Additional context**
Initially discovered due to incorrect ownership/permissions on `~/.vnc`

**Comment by CendioOssman** [comment]
I don't think it is possible for us to print any error message back through systemctl, unfortunately.

What might be possible is to report a general failure starting the service. I think systemctl will print something generic at that point.

It's unfortunately not trivial, though, as we are pretty deep in to the startup when we check for the password file.
